### DISCUSSION TITLE
Why do we need a test case for data-bs-toggle="dropdown" ?

### DISCUSSION THREAD

**Discussion by sijusamson** [twbs/bootstrap#32613] [Q&A] [answered]
In bootstrap documentation, it is clearly mentioned that we need to add `data-bs-toggle="dropdown"`, no matter if we call via JavaScript or data-api, so why do we need a testcase [in dropdown](https://github.com/twbs/bootstrap/blob/main/js/tests/unit/dropdown.spec.js) **that it should add a listener on trigger which do not have data-bs-toggle="dropdown"**.
Adding screenshots of documentation and code for reference.

**_Documentation_**
![Screenshot from 2020-12-25 20-06-12](https://user-images.githubusercontent.com/38852547/103137244-34aeda80-46ed-11eb-85c3-d846cf7b1c7a.png)

**_Test Case_**
![Screenshot from 2020-12-25 20-09-48](https://user-images.githubusercontent.com/38852547/103137245-37113480-46ed-11eb-9725-d2ed605dee4c.png)

**Comment by WanielDeiss** [comment, marked as answer]
The test ist needed for users they don't want to setup the bs dropdown automatically with the data-api.

The data-api `data-bs-toggle="dropdown"` is fine for users with static websites and users who serve the whole dom at once.
But for user who renders dynamically there html, need other ways to initialise the functionality. And for that people, is the test.

**Reply by sijusamson** [reply]
Thank you so much for your answer @WanielDeiss
